share/pot/common.sh

```diff
@@ -698,6 +698,17 @@ _is_natural_number()
 	esac
 }
 
+# $1 a string
+# tested ( common8 )
+_contains_spaces()
+{
+	if [ "$1" = "${1% *}" ]; then
+		return 1 # false
+	else
+		return 0 # true
+	fi
+}
+
```

Comment:
Alternative implementation that will catch all kind of whitespace characters (like `\t \v \f \r`) :

```sh
_contains_whitespace()
{
    echo "$1" | grep -q "[[:space:]]"
}
```

Reply:
yep, your implementation is more generic

Reply:
I used whitespace, as it means space, tab etc., while space imho refers to ascii 32 (even though the regex is `[[:space:]]`). Anyway :)